amundsen_application/static/css/_icons.scss

```diff
@@ -11,11 +11,11 @@ img.icon {
   width: 24px;
 
   &.icon-color {
-    background-color: $brand-color-3;
+    background-color: $icon-bg-color;
```

Comment:
nit: I found this one confusing at first in terms of the difference between `icon-bg` and `icon-bg-color` since they're all background colors. But then I realized this one is the icon color that matches the "brand color", so perhaps this could be called `icon-bg-brand`.

Reply:
Will update.